tests/load/athena_iceberg/test_athena_adapter.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+import dlt
+from dlt.destinations import filesystem
+from dlt.destinations.impl.athena.athena_adapter import athena_adapter, athena_partition
+from tests.load.utils import destinations_configs, DestinationTestConfiguration
+
+
+def test_iceberg_partition_hints():
```

Comment:
should we use `destination_config` to filter for `athena` so we run it only when athena tests run on CI?

Reply:
The whole package is skipped by: https://github.com/dlt-hub/dlt/blob/athena-partitions/tests/load/athena_iceberg/__init__.py#L1-L4